api/internal/handler/ssl/ssl.go

```diff
@@ -147,6 +148,8 @@ func (h *Handler) Create(c droplet.Context) (interface{}, error) {
 	}
 
 	ssl.ID = input.ID
+	//set default value for SSL status, if not set, it will be 0 which means disable.
+	ssl.Status = conf.SSLDefaultStatus
 	if err := h.sslStore.Create(c.Context(), ssl); err != nil {
 		return handler.SpecCodeResponse(err), err
```

Comment:
Did we catch this error and write it to the error log?  please confirm it. if not, we can create a new issue to fix it

Reply:
sure, I will create an issue.

Reply:
if you created, you should better leave an issue id here.